drafttopic/utilities/wikiprojects_common.py

```diff
@@ -4,7 +4,7 @@
 logger = logging.getLogger(__name__)
 
 
-WIKIPROJECT_FETCH_THREADS = 4
+WIKIPROJECT_FETCH_THREADS = 1000
```

Comment:
This seems hazardous.

Reply:
lol I agree.  It seems to work well, though, at about 15% of all CPUs or something, and not thrashing the "disc".

Reply:
I think this violates the wikimedia API ToS.  Sending 1000 parallel requests seems like bad citizen behavior.

Reply:
Confirmed that no one cares :+1: